src/index.ts

```diff
@@ -838,7 +841,7 @@ class ForkTsCheckerWebpackPlugin {
           (this.lints || []).concat(this.diagnostics).forEach(message => {
             const formattedMessage = this.formatter(message, this.useColors);
 
-            message.isWarningSeverity()
+            message.isWarningSeverity() && !this.ignoreLintWarnings
               ? this.logger.warn(formattedMessage)
```

Comment:
I think this might be an issue in the case where you have a warning and you are ignoring warnings - it looks like an error would be logged in that case ..

Reply:
ah. So int that case I would want to do nothing here, right?

Reply:
Yeah I think so

Reply:
Ok, updated. Let me know if that looks correct to you.